src/plugins/data/public/query/filter_manager/lib/mappers/map_phrase.ts

```diff
@@ -26,6 +31,13 @@ export function getPhraseDisplayValue(
   fieldType?: string
 ): string {
   const value = filter.meta.value ?? filter.meta.params?.query;
+
+  if (isEmptyString(value)) {
+    return i18n.translate('data.filter.filterBar.emptyString', {
+      defaultMessage: 'Empty string',
+    });
+  }
+
```

Comment:
Hi @lgestc,

In Discover grid, doc viewer and filters we already show it as `(empty)`. We would like to avoid inconsistency in UI views. Would it be possible to align?

<img width="600" alt="Screenshot 2024-04-10 at 16 22 18" src="https://github.com/elastic/kibana/assets/1415710/c7ae6ad7-ea82-430b-86ea-2f991fec3991">